ruby/computer_science/time_complexity.md

```diff
@@ -141,7 +141,7 @@ Do the number of steps matter? Yes, they might. We'll touch on when this may be
 
 Logarithmic Complexity tells us that the numbers of steps an algorithm takes increases by 1 as the data doubles. That's still pretty efficient when you think about it. Going from 5,000 to 10,000 data elements and only taking one additional step can scale really well.
 
-One such algorithm that does this is Binary Search. It only works on sorted arrays but if you have an array of 10 items in sorted order 
+One such algorithm that does this is Binary Search. It only works on sorted arrays but if you have an array of 10 items in sorted order
```

Comment:
Hey @stxss, thanks for contributing, there doesn't appear to be a typo fix in this PR. The only change is this space removed here.

[A fix](https://github.com/TheOdinProject/curriculum/pull/25184) was merged for that typo recently which is probably making the changes you made in this branch not show up.

Reply:
Duly noted, sorry for this PR not being useful.

Reply:
No need to apologise, the other PR just got there first, happens all the time.